AdminWebsite/AdminWebsite/ClientApp/src/app/booking/assign-judge/assign-judge.component.html

```diff
@@ -115,7 +115,7 @@ <h2 class="govuk-error-summary__title" id="error-summary-title">
         />
       </div>
 
-      <div *ngIf="showStaffMemberFeature">
+      <!-- <div *ngIf="showStaffMemberFeature">
```

Comment:
Please remove the code instead of commenting out. We have source control if we ever need to undo this change or know what it looked like historically